src/main/java/org/schemaspy/DbDriverLoader.java

```diff
@@ -148,13 +157,34 @@ protected Driver getDriver(String driverClass, String driverPath) throws Malform
         return driver;
     }
 
+    private void loadAdditionalJarsForDriver(String driverPath, Set<URL> classpath) throws MalformedURLException {
+        File driverFolder = new File(Paths.get(driverPath).getParent().toString());
+
+        if (driverFolder != null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)